sdk/tests/conformance2/reading/read-pixels-pack-parameters.html

```diff
@@ -56,6 +65,45 @@
     }
 }
 
+function paintWebGLCanvas(gl)
+{
+    var canvas2d = document.getElementById("canvas2d");
+    var ctx2d = canvas2d.getContext("2d");
+    for (var row = 0; row < 4; ++row) {
+        ctx2d.fillStyle = "rgba(" + expectedColor[row][0] + ", " +
+            expectedColor[row][1] + ", " + expectedColor[row][2] + ", " +
+            expectedColor[row][3] + ")";
+        ctx2d.fillRect(0, row, 4, 1);
+    }
+
+    var program = wtu.setupTexturedQuad(gl);
+    gl.disable(gl.DEPTH_TEST);
+    gl.disable(gl.BLEND);
+
+    var tex = gl.createTexture();
+    gl.bindTexture(gl.TEXTURE_2D, tex);
+    gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA, gl.RGBA, gl.UNSIGNED_BYTE, canvas2d);
+    gl.generateMipmap(gl.TEXTURE_2D);
```

Comment:
Do we really need mipmaps? This texture shouldn't be down- or upsampled; it should use NEAREST filtering.